students/Kate_Koehn/Lesson08/inventory.py

```diff
@@ -0,0 +1,54 @@
+'''
+To summarize: 1. You will create a program to initially create, and subsequently update, a CSV file that lists
+which furniture is rented to which customer (to replace use of the spreadsheet mentioned above).
+2. You will create additionally functionality that will load individual customers rentals.
+'''
+
+import csv
+from functools import partial
+
+
+def add_furniture(invoice_file="", customer_name="", item_code="", item_description="", item_monthly_price=0.00):
+    with open(invoice_file, 'a+') as csv_file:
+        writer = csv.writer(csv_file)
+        row = customer_name, item_code, item_description, item_monthly_price
+        writer.writerow(row)
+
+
+def test_add_data():
+    """
+    function to add test data using add_furniture function
+    """
+    add_furniture("invoice_file.csv", "Elisa Miles", "LR04", "Leather Sofa", 25.00)
+    add_furniture("invoice_file.csv", "Edward Data", "KT78", "Kitchen Table", 10.00)
+    add_furniture("invoice_file.csv", "Alex Gonzales", "BR02", "Queen Mattress", 17.00)
+
+
+
+def single_customer(customer_name, customer_file):
+    """
+    function to iterate through rental file and add a single customer to the inventory file
+    """
+    def add_rentals(rental_file):
+        with open(customer_file) as rental_csv:
+            reader = csv.reader(rental_csv)
+
+            add_item = partial(add_furniture, customer_name=customer_name, invoice_file=rental_file)
+
+            for row in reader:
+                add_item(item_code=row[1], item_description=row[2], item_monthly_price=row[3])
+
+    return add_rentals
```

Comment:
Closure looks great. Remember in the doc string to be explicit that it will return a function. Functions like this should be very explicit, outer function takes these variables, returns a function that takes these variables.